src/locale/locales/ca/messages.po

```diff
@@ -0,0 +1,2489 @@
+msgstr "Afegeix una targeta a l'enllaç:"
+
+#: src/view/com/modals/ChangeHandle.tsx:415
+msgid "Add the following DNS record to your domain:"
+msgstr "Afegeix el següent registre DNS al teu domini:"
+
+#: src/view/com/profile/ProfileHeader.tsx:353
+msgid "Add to Lists"
+msgstr "Afegeix a les llistes"
+
+#: src/view/screens/ProfileFeed.tsx:271
+msgid "Add to my feeds"
+msgstr "Afegeix als meus canals"
+
+#: src/view/com/modals/ListAddRemoveUsers.tsx:191
+#: src/view/com/modals/UserAddRemoveLists.tsx:128
+msgid "Added to list"
+msgstr "Afegit a la llista"
+
+#: src/view/screens/PreferencesHomeFeed.tsx:164
+msgid "Adjust the number of likes a reply must have to be shown in your feed."
+msgstr "Ajusta el nombre de m'agrades que hagi de tenir una resposta per aparèixer al teu canal."
+
+#: src/view/com/modals/SelfLabel.tsx:75
+msgid "Adult Content"
+msgstr "Contingut per a adults"
+
+#: src/view/screens/Settings.tsx:569
+msgid "Advanced"
+msgstr "Avançat"
+
+#: src/view/com/composer/photos/Gallery.tsx:130
+msgid "ALT"
+msgstr "ALT"
+
+#: src/view/com/modals/EditImage.tsx:315
+msgid "Alt text"
+msgstr "Text alternatiu"
+
+#: src/view/com/composer/photos/Gallery.tsx:209
+msgid "Alt text describes images for blind and low-vision users, and helps give context to everyone."
+msgstr "El text alternatiu descriu les imatges per a les persones cegues o amb problemes de visió, i ajuda a donar context a tothom."
+
+#: src/view/com/modals/VerifyEmail.tsx:118
+msgid "An email has been sent to {0}. It includes a confirmation code which you can enter below."
+msgstr "S'ha enviat un correu a {0}. Inclou un codi de confirmació que has d'entrar aquí sota."
+
+#: src/view/com/modals/ChangeEmail.tsx:119
+msgid "An email has been sent to your previous address, {0}. It includes a confirmation code which you can enter below."
+msgstr "S'ha enviat un correu a la teva adreça prèvia, {0}. Inclou un codi de confirmació que has d'entrar aquí sota."
+
+#: src/view/com/notifications/FeedItem.tsx:237
+#: src/view/com/threadgate/WhoCanReply.tsx:178
+msgid "and"
+msgstr "i"
+
+#: src/view/screens/LanguageSettings.tsx:95
+msgid "App Language"
+msgstr "Idioma de l'aplicació"
+
+#: src/view/screens/Settings.tsx:589
+msgid "App passwords"
+msgstr "Contrasenyes de l'aplicació"
+
+#: src/view/screens/AppPasswords.tsx:186
+msgid "App Passwords"
+msgstr "Contrasenyes de l'aplicació"
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:236
+msgid "Appeal content warning"
+msgstr "Afegeix una advertència de contingut"
+
+#: src/view/com/modals/AppealLabel.tsx:65
+msgid "Appeal Content Warning"
+msgstr "Afegeix una advertència de contingut"
+
+#: src/view/com/util/moderation/LabelInfo.tsx:52
+msgid "Appeal this decision"
+msgstr "Apel·lar aquesta decisió"
+
+#: src/view/com/util/moderation/LabelInfo.tsx:56
+msgid "Appeal this decision."
+msgstr "Apel·lar aquesta decisió."
+
+#: src/view/screens/Settings.tsx:432
+msgid "Appearance"
+msgstr "Aparença"
+
+#: src/view/screens/AppPasswords.tsx:223
```

Comment:
I wouldn't have used "tu" to talk to the user.
In addition it may be better to avoid a masculine form, segur.
«Voleu vertaderament eliminar la contrasenya de l'aplicació ?» seems to me more universal :)

Reply:
Seems fair, I changed the "tu" to "vos" to talk to the user (Softcatalà conventions), I leave "Segur" as it is an adverb in the form "Segur que vols eliminar...". Thanks for the suggestions.